lib/rubocop/ast/node.rb

```diff
@@ -314,13 +314,18 @@ def parent_module_name
       ## Predicates
 
       def multiline?
-        source_range && (source_range.first_line != source_range.last_line)
+        line_count > 1
       end
 
       def single_line?
         !multiline?
```

Comment:
I think we can express this as `line_count.one?` now. Since if `source_range` for some reason is `nil`, that isn't a single-line node. WDYT?

Reply:
While `Numeric` has the `zero?` method, `one?` doesn't seem native to the Ruby lib. How about `line_count == 1`?

Reply:
Right. Sorry. `#one?` is part of `Enumerable`. 🙂 `line_count == 1` is fine.